src/ddmd/func.d

```diff
@@ -739,6 +739,10 @@ extern (C++) class FuncDeclaration : Declaration
 
             default:
                 {
+                    // Bugzilla 16273
```

Comment:
Didn't I see a pr recently that made these comments url links? Not bothered one way or the other myself.